src/config/routes.ts

```diff
@@ -1,34 +1,33 @@
 export default {
-  index: () => '/',
-  searchOverlay: () => `/search`,
-  search: ({ query }: { query?: string } = {}) => {
-    const basePath = '/search'
+  viewer: {
+    index: () => '',
+    searchOverlay: () => `search`,
+    search: ({ query }: { query?: string } = {}) => {
+      const basePath = 'search'
 
-    if (query) {
-      const searchQueryParams = new URLSearchParams()
-      searchQueryParams.set(QUERY_PARAMS.SEARCH, query.trim())
-      return `${basePath}?${searchQueryParams.toString()}`
-    }
+      if (query) {
+        const searchQueryParams = new URLSearchParams()
+        searchQueryParams.set(QUERY_PARAMS.SEARCH, query.trim())
+        return `${basePath}?${searchQueryParams.toString()}`
+      }
 
-    return basePath
+      return basePath
+    },
+    channel: (id = ':id') => `channel/${id}`,
+    channels: () => 'channels',
+    video: (id = ':id') => `video/${id}`,
+    videos: () => 'videos',
+  },
+  studio: {
+    index: () => 'studio',
+    newChannel: () => 'channel/new',
+    editChannel: () => 'channel',
+    videos: () => 'videos',
+    uploads: () => 'uploads',
+  },
+  playground: {
+    index: () => 'playground',
```

Comment:
We could list playground paths here like in the studio to keep consistency

Reply:
I'd like playground to be "self-contained" so I wouldn't want to leak its routes externally